api/src/identity-access-management/domain/usecases/get-user-details-by-user-ids.usecase.js

```diff
@@ -1,3 +1,10 @@
-export const getUserDetailsByUserIds = function ({ userIds, userRepository }) {
-  return userRepository.getByIds(userIds);
+export const getUserDetailsByUserIds = async function ({ userIds, userRepository }) {
+  const activeUsers = [];
+  const users = await userRepository.getByIds(userIds);
+
+  users.forEach((user) => {
+    if (!user.isAnonymousUser) activeUsers.push(user);
+  });
+
+  return activeUsers;
```

Comment:
Par rapport au commentaire précédent:

```suggestion
  const users = await userRepository.getByIds(userIds);
  return users.filter((user) => user.isActive());
```